crates/router/src/core/payments/flows.rs

```diff
@@ -142,6 +142,7 @@ impl<const T: u8>
 }
 
 default_imp_for_complete_authorize!(
+    connector::Payone,
```

Comment:
This should be added in alphabetical order

Reply:
done